api/reaction.go

```diff
@@ -112,23 +111,6 @@ func deleteReaction(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
-func sendReactionEvent(event string, channelId string, reaction *model.Reaction, post *model.Post) {
-	// send out that a reaction has been added/removed
-
-	message := model.NewWebSocketEvent(event, "", channelId, "", nil)
-	message.Add("reaction", reaction.ToJson())
-	app.Publish(message)
-
-	// THe post is always modified since the UpdateAt always changes
-	app.Global().InvalidateCacheForChannelPosts(post.ChannelId)
-	post.HasReactions = true
-	post.UpdateAt = model.GetMillis()
-	umessage := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_POST_EDITED, "", channelId, "", nil)
-	umessage.Add("post", post.ToJson())
-	app.Publish(umessage)
-
-}
```

Comment:
Wasn't referenced anywhere.

Reply:
Looks like it was superseded by app.sendReactionEvent and never removed